ex2_system/source/beacon/beacon_task.c

```diff
@@ -46,6 +97,7 @@ static void *beacon_daemon(void *pvParameters) {
     /* Constructing the system beacon content */
     // Refer to table 3 of MOP
     UHF_configStruct beacon_msg;
+    beacon_msg.len = 0xBC;

```

Comment:
This should be the size of the contents in the packet we're sending. UHF will give out an error if this is set to anything more than 0x61. Something like `sizeof(beacon_packet_x)` would work but that really needs careful programming for alternating between the two beacon packets.

Reply:
This wasn't supposed to make it here. will remove